### ISSUE TITLE
Release code for ONG on Hugging Face

### ISSUE THREAD

**Issue by NielsRogge** [yajatyadav/orthogonal-natural-gradient#1]
Hi @yajatyadav 🤗

I'm Niels and work as part of the open-source team at Hugging Face. I discovered your work on Arxiv and was wondering whether you would like to submit it to [hf.co/papers](https://hf.co/papers) to improve its discoverability. If you are one of the authors, you can submit it at https://huggingface.co/papers/submit.

The paper page lets people discuss about your paper and lets them find artifacts about it (your models for instance),
you can also claim the paper as yours which will show up on your public profile at HF, add Github and project page URLs.

I noticed in the abstract of your paper "ONG: Orthogonal Natural Gradient Descent" (arXiv:2508.17169) that you mentioned the code for your experiments/reproducibility is available at https://github.com/yajatyadav/orthogonal-natural-gradient. However, the README in that repository states it is for an older paper (arXiv:2006.11942).

It would be great if the code and/or any pre-trained model checkpoints for "ONG" could be made available on the 🤗 hub, to improve their discoverability/visibility.
We can help you link them to your paper page and add relevant tags.

If you're interested in hosting the code/checkpoints on https://huggingface.co/models, here's a guide: https://huggingface.co/docs/hub/models-uploading.

You can also build a demo for your model on [Spaces](https://huggingface.co/spaces), we can provide you a ZeroGPU [grant](https://huggingface.co/docs/hub/en/spaces-gpus#community-gpu-grants), which gives you A100 GPUs for free.

Let me know if you're interested/need any guidance, or if there's a different URL for the "ONG" code :)

Kind regards,

Niels
